ArkEcosystem/explorer
# feat: implement sentry for error reporting

## Summary

https://app.clickup.com/t/1kama3p

Adds sentry

**To test:**

- Add a valid sentry dsn on the `.env`file `SENTRY_LARAVEL_DSN=`
- create an exception and see if is reported

**To deploy:**
- Don't forget to define `SENTRY_LARAVEL_DSN`

## Checklist
